packages/libs/pactjs-test-project/src/transfer.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  IUnsignedTransaction,
+  Pact,
+  signWithChainweaver,
+} from '@kadena/pactjs';
+
+const sender: string = 'croesus';
+const receiver: string =
+  'k:554754f48b16df24b552f6832dda090642ed9658559fef9f3ee1bb4637ea7c94';
+// const sender: string = receiver;
+const senderNoK: string =
+  '2993f795d133fa5d0fd877a641cabc8b28cd36147f666988cacbaa4379d1ff93';
+// const senderNoK: string = sender.split('k:')[1];
+const amount: number = 1000.0;
```

Comment:
`PactNumber` can be used here

Reply:
This is the consuming code. I'm not really a fan of using the complex types in the consuming code. I'd like the level of entry to be as low as possible. We now dynamically convert number into PactNumber. See also https://github.com/kadena-community/kadena.js/pull/59#discussion_r964748673